PetShare/Configuration/PaginationConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace PetShare.Configuration
+{
+    public sealed class PaginationConfiguration
+    {
+        public const string SectionName = "Pagination";
+
+        [Required]
+        public required int DefaultPageSize { get; init; }
+        [Required]
+        public required int DefaultPageNumber { get; init; }
```

Comment:
This is because of the specification requirements. If null values are passed in request query then some default parameters should be used. I wanted to keep it transparent and easy to configure later, hence the confiuration.